frontend/src/components/MapView/Layers/PointDataLayer/index.tsx

```diff
@@ -142,7 +137,7 @@ function PointDataLayer({ layer, before }: LayersProps) {
   }
   return (
     <GeoJSONLayer
-      before={before}
+      // before={before}
```

Comment:
@wadhwamatic would you like the points to appear above or below the admin boundaries?